sites/partners/src/listings/PaperListingDetails/sections/DetailLeasingAgent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react"
+import { t, GridSection, ViewItem, GridCell } from "@bloom-housing/ui-components"
+import { ListingContext } from "../../ListingContext"
+
+const DetailLeasingAgent = () => {
+  const listing = useContext(ListingContext)
+
+  return (
+    <GridSection
+      className="bg-primary-ligher"
+      title={t("listings.sections.leasingAgentTitle")}
+      grid={false}
+      inset
+    >
+      <GridSection columns={3}>
+        <GridCell>
+          <ViewItem label={t("leasingAgent.name")}>{listing.leasingAgentName}</ViewItem>
+        </GridCell>
+        <GridCell>
+          <ViewItem label={t("t.email")}>{listing.leasingAgentEmail}</ViewItem>
+        </GridCell>
+        <GridCell>
+          <ViewItem label={t("t.phone")}>{listing.leasingAgentPhone}</ViewItem>
+        </GridCell>
+      </GridSection>
```

Comment:
This is just something for us to think about. It's looking like we might benefit from a component that could make these go a little faster. It could  take something like 
`{ columns: number, fields: { label: string, value: T }[] }[]`